scripts/common.sh

```diff
@@ -10,4 +10,25 @@ log(){
   else
     echo "$(date '+%Y-%m-%d %H:%M:%S') [INFO] $1"
   fi
+}
+
+# Fetch the value of requested variable defined in config file
+# params: $1 - variable name, $2 - config file full path
+get_variable(){
+  if [ "$#" -eq 2 ]; then
+    if [ ! -f "${2}" ]; then
+      echo "File ${2} is not existed."
+      return 1
+    fi
+    local VALUE=$(grep -o '^[^#]*' "${2}" | grep "${1}" | sed 's/ //g' | grep "${1}=" | sed -nE 's/^.*"(.*)".*$/\1/p')
+    if [ ! $(echo "${VALUE}" | wc -l) -eq 1 ];then
+      echo "ERROR - '${1}' is re-defined in '${2}'";
+      echo "${VALUE}"
+      return 1;
+    fi
+    echo "${VALUE}"
```

Comment:
```suggestion
    log "${1}: ${VALUE}"
```

Reply:
We should not put in log this line. This is actually the output of the function. This should be the absolute value of the given variable in the config file and this will be used by the caller.